Notes/Algorithms/Data Structures/Hash-Tables.c

```diff
@@ -4,34 +4,231 @@
 
 #define SUCCESS 0
 #define ERROR -1
-#define OVERFLOW -2
-#define EMPTY -3
+
+#define TEST 9
+#define EXPECTED 10
+
+#define SIZE 10
```

Comment:
As discussed in a couple other places, these 3 globals should be converted into local state.  The api of your insert and delete functions will need to adjust to accept a pointer to an array of Items.

Reply:
Updated in [commit a0219c38606fafc63d5c641e0623442f105ff2b](https://github.com/kcarrel/Training-Notes/pull/12/commits/a0219c38606fafc63d5c641e0623442f105ff2b7) to move away from global state/magic numbers.